perfkitbenchmarker/linux_packages/cuda_toolkit_8.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Module containing CUDA toolkit 8 installation and cleanup functions."""
+
+from perfkitbenchmarker import regex_util
+
+CUDA_TOOLKIT_UBUNTU = 'cuda-repo-ubuntu1604_8.0.44-1_amd64.deb'
+CUDA_TOOLKIT_UBUNTU_URL =\
+    'http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/%s' %\
```

Comment:
I just tried with Ubuntu 14.04 and the 14.04 CUDA package. It failed due to a driver issue. I don't want to spend anymore time on it right now. If we have a strong case for testing on Ubuntu 14.04 it will take some work to get the correct driver installed and running correctly. Everything works fine on 16.04.